hudi-common/src/main/java/org/apache/hudi/common/config/HoodieStorageConfig.java

```diff
@@ -170,6 +170,12 @@ public class HoodieStorageConfig extends HoodieConfig {
       .withDocumentation("Expected additional compression as records move from log files to parquet. Used for merge_on_read "
           + "table to send inserts into log files & control the size of compacted parquet file.");
 
+  public static final ConfigProperty<String> HOODIE_AVRO_WRITE_SUPPORT_CLASS = ConfigProperty
+      .key("hoodie.avro.write.support.class")
+      .defaultValue("org.apache.hudi.avro.HoodieAvroWriteSupport")
```

Comment:
Since this is advanced, could you mark this with `.markAdvanced()`?

Reply:
Yes, added markAdvanced to the config.